client/js/components/procedure/admin/DpNewProcedure/DpNewProcedure.vue

```diff
@@ -7,53 +7,288 @@
   All rights reserved
 </license>
 
+<template>
+  <form
+    name="xsubmititem"
+    :action="Routing.generate('DemosPlan_procedure_new')"
+    enctype="multipart/form-data"
+    method="post"
+    data-dp-validate>
+    <input
+      type="hidden"
+      name="_token"
+      :value="token">
+    <input
+      type="hidden"
+      name="action"
+      value="new">
+    <input
+      type="hidden"
+      name="r_master"
+      value="false">
+    <!-- allow publishing of Statements by default -->
+    <input
+      type="hidden"
+      name="r_publicParticipationPublicationEnabled"
+      value="1">
+    <div v-if="hasPermission('feature_use_plis')">
+      <input
+        type="hidden"
+        name="r_name"
+        value="">
+      <input
+        type="hidden"
+        name="r_externalDesc"
+        value="">
+      <input
+        type="hidden"
+        name="r_mapExtent"
+        value="">
+    </div>
+    <fieldset>
+      <addon-wrapper hook-name="bimschg.request" />
+      <div v-if="hasPermission('feature_use_plis')">
+        <dp-form-row>
+          <dp-select
+            id="r_plisId"
+            :label="{ text: Translator.trans('name'), hint: Translator.trans('explanation.plis.procedurename') }"
+            name="r_plisId"
+            :options="plisNameOptions" />
+        </dp-form-row>
+        <dl class="u-nojs-hide--block">
+          <dt
+            v-text="Translator.trans('public.participation.desc')"
+            class="weight--bold" />
+          <dd
+            id="js__plisPlanungsanlass"
+            class="list-style-none" />
+        </dl>
+      </div>
+      <dp-form-row v-else>
+        <dp-input
+          data-cy="newProcedureTitle"
+          id="r_name"
+          :label="{ text: Translator.trans('name') }"
+          :maxlength="200"
+          name="r_name"
+          required
+          type="text" />
+      </dp-form-row>
+      <!--      {% block master %}-->
+      <dp-form-row v-if="hasPermission('feature_procedure_templates')">
+        <dp-select
+          id="blueprint"
+          :label="{
+            hint: blueprintHint,
+            text: Translator.trans('master')
+          }"
+          name="r_copymaster"
+          :options="blueprintOptions"
+          :selected="masterBlueprintId"
+          @select="setBlueprintData" />
+      </dp-form-row>
+
+<!--      {# Only show select if there is more than one choice. Otherwise, pass the id as the value of a hidden field. #}-->
```

Comment:
nitpick: Use the correct indentation and:
```suggestion
<!-- Only show select if there is more than one choice. Otherwise, pass the id as the value of a hidden field. -->
```